test/jasmine/tests/lib_date_test.js

```diff
@@ -222,6 +222,68 @@ describe('dates', function() {
                 expect(Lib.ms2DateTime(Lib.dateTime2ms(v[0]), v[1])).toBe(v[2], v);
             });
         });
+
+        it('should work right with inputs beyond our precision', function() {
```

Comment:
worked fine *before* all this crap with regular dates... during my work it broke with regular and world dates, now it works for all.